google/ggrc-core
# GGRC-8614 Replace can-map with can-define-map for tree-widget-container component

# Issue description

Actually can-map, can-list are used. can-map, can-list are legacy and the alternatives in latest versions of CanJS are DefineMap, DefineList.

# Steps to test the changes

Open the APP and check Tree-View component on all pages (My Work page, All objects, My Assessments, Tree-View in scope objects (in scope Audit, Program, etc.))

# Sanity checklist




# PR Review checklist
